### ISSUE TITLE
Some issues to fix

### ISSUE THREAD

**Issue by mgland** [mgland/iutest#12]
Error when no tests were collected for pyunit.
```
# Traceback (most recent call last):
#   File "/scratch/miguelg/packages/iutest/0.1.4/python-2.7/python/iutest/ui/iutestwindow.py", line 377, in _onTestRunnerSwitched
#     self._onReloadUiButtonClicked()
#   File "/scratch/miguelg/packages/iutest/0.1.4/python-2.7/python/iutest/ui/iutestwindow.py", line 447, in _onReloadUiButtonClicked
#     self._applyCurrentFilter(removeStateFilters=True, keepUiStates=True)
#   File "/scratch/miguelg/packages/iutest/0.1.4/python-2.7/python/iutest/ui/iutestwindow.py", line 699, in _applyCurrentFilter
#     self._applyFilterTextWithState(searchText, keepUiStates=keepUiStates)
#   File "/scratch/miguelg/packages/iutest/0.1.4/python-2.7/python/iutest/ui/iutestwindow.py", line 726, in _applyFilterTextWithState
#     self._view.setFilterKeywords(keywords, ensureFirstMatchVisible=not keepUiStates)
#   File "/scratch/miguelg/packages/iutest/0.1.4/python-2.7/python/iutest/ui/unittesttree.py", line 211, in setFilterKeywords
#     for index in range(self._rootTestItem.childCount()):
# AttributeError: 'NoneType' object has no attribute 'childCount'
```
Both pyunit and nose2 need to issue a clear report on that.

**Comment by mgland** [comment]
Need to report an error when unable to launch goToCode.

**Comment by mgland** [comment]
Deduplication for selected test on module or package not works now for nose2

**Comment by mgland** [comment]
These are all fixed in 0.1.5
